js/piwik.js

```diff
@@ -2516,6 +2582,7 @@ if (typeof Piwik !== 'object') {
              * Send request
              */
             function sendRequest(request, delay, callback) {
+                lastTrackerRequestTime = (new Date()).getTime();
```

Comment:
Think this is a good idea, results in less code. I think there won't be any issues due to event queue, but tests should catch them if there are.